configuration/conceptclasses/orderableclasses.csv

```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Name,Description
+9dc243d8-9b36-48b7-99e3-211c1b193692,,Product,Concept class for Product orders

```

Comment:
Should this file be part of the PR?

Reply:
Some of the products were submitted via the drugs...this is why i added the product related items in this PR too